src/.sonarlint/project-monai_monai-deploy-informatics-gateway/CSharp/SonarLint.xml

```diff
@@ -76,7 +76,7 @@
       <Key>S3776</Key>
       <Parameters>
         <Parameter>
-          <Key>propertyThreshold</Key>
+          <Key>propertyThreshold </Key>
```

Comment:
accidental space?

Reply:
Interesting, these were pulled back from sonarcloud. lol - I'll remove the space